test/support/page_objects/header_page.js

```diff
@@ -47,7 +47,7 @@ export default class HeaderPage {
   }
 
   isTimepickerOpen() {
-    return this.remote.setFindTimeout(defaultFindTimeout)
+    return this.remote.setFindTimeout(2000)
```

Comment:
Yes.  When the timepicker is open we find it right away.  When it's not open we don't want to wait the 10 second defaultFindTimeout.